cleanup.sh

```diff
@@ -9,12 +9,24 @@ read
 SRC_DIR="$(dirname "$(realpath -s "${0}")")"
 
 cd "${SRC_DIR}"
-rm   -rf .git build config.h doc
-find . -name CMakeLists.txt.user -delete
-find . -name CMakeCache.txt -delete
-find . -name cmake_install.cmake -delete
+# rm -rf .git build config.h doc
```

Comment:
this cleanup script is supposed to be executed as a part of release process and not regular user workflow, as we don't wish to have `.git` and build garbage to be packaged into release archives.